admin/workspace-providers/deployment/ec2.md

```diff
@@ -0,0 +1,69 @@
+---
+title: EC2
+description: Learn how to deploy a workspace provider to an EC2 cluster.
+---
+
+This article walks you through the process of deploying a workspace provider to
+an EC2 instance.
+
+## Prerequisites
+
+You must have an
+[**AWS access key ID** and **secret access key**](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys).
+
+We recommend having the [AWS CLI](https://aws.amazon.com/cli/) installed and
+configured as well.
+
+## Creating a workspace provider
+
+1. Log into Coder as a site manager, and go to **Manage** > **Workspace
+   providers**.
+
+1. In the top-right next to **Create Kubernetes Provider**, click on the **down
+   arrow** and select **Create Amazon EC2 Provider**.
+
+1. Provide a **name** to identify the provider.
+
+1. Provide the requested configuration details to connect Coder to your AWS
+   account:
+
+   - **Access key ID**: the AWS access key associated with your account
+   - **Secret access key**: the AWS secret access key associated with your
+     account
+   - **AWS region ID**: The AWS region where the EC2 instances should be created
+   - **AWS availability zone**: The AWS availability zone associated with the
```

Comment:
Do we have additional guidance for choosing one? The CLI snippet is one I found while Googling, but I notice the ones we use in master have `a` -- what does that mean?